pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/TopicsImpl.java

```diff
@@ -1231,6 +1238,42 @@ public void failed(Throwable throwable) {
         return future;
     }
 
+    @Override
+    public CompletableFuture<Void> triggerOffloadAsync(String topic, long sizeThreshold) {
+        CompletableFuture<Void> future = new CompletableFuture<>();
+        try {
+            PersistentTopicInternalStats stats = getInternalStats(topic);
```

Comment:
Please use `getInternalStatsAsync` instead of `getInternalStats` to avoid thread blocking.

Reply:
@nodece please review again bc9d73554a195d38e5d5704bdbae7e1c9eb0cfc0